src/Cronofy.php

```diff
@@ -845,6 +845,18 @@ public function realTimeScheduling($params)
         return $this->httpPost("/" . self::API_VERSION . "/real_time_scheduling", $postFields);
     }
 
+    public function realTimeSchedulingDisable($params)
```

Comment:
2/10 - I'd pop this function after the `realTimeSequencing` function. It just makes a bit more sense if reading the whole file to see the creation function before the disable.

Reply:
Probably needs to be called `disableRealTimeScheduling` too - looks like we usually name things `<verb><noun>` such as `createEvent`